config/initializers/01_indexing_adapter_initializer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```

Comment:
The rename to `01_...` is not needed.  All gem initializers are run before running application initializers.  The problem was with the `to_prepare` in the old version of this initializer.  It was causing the block within the `to_prepare` to run after all initializers completed.

I recommend keeping the original name and simply move the code in the `to_prepare` block outside.